### PR TITLE
Frontend updates: welcome view and project view

### PR DESCRIPTION
**Pull request by akuleshov7**
### What's done:
- updated welcome view animation
- updated error and warning logic for project vie

https://user-images.githubusercontent.com/58667063/184942326-d6b0933f-84f9-4ac5-8620-0f0790cbd887.mov

### CODE REVIEW

**Comment by sanyavertolet** [comment]
After I took a look at how different browsers (firefox, chrome and safari) draw alert or confirmation windows, I started to like an idea of moving to browser modal windows in some cases.

**Comment by akuleshov7** [comment]
> After I took a look at how different browsers (firefox, chrome and safari) draw alert or confirmation windows, I started to like an idea of moving to browser modal windows in some cases.

I think we should give a try for it and later compare how it works

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ProjectView.kt [hunk 1]
```diff
@@ -455,19 +424,6 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
         submitRequest("/submitExecutionRequest", Headers(), formData)
     }
 
-    private fun submitExecutionRequestByContest() {
```
Comment (akuleshov7) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ProjectView.kt:458]: very strange resolving of conflicts by idea

**Review by sanyavertolet — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ProjectView.kt (hunk 1 above)
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ProjectView.kt:458]: done

**Pull request merged**